# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1323171538 -46800
# Node ID 8216b40e074d50adc96ae3d979ba5ff275fe025f
# Parent  a6dff25c1409e926b6b2f1622c77987f08659203
Add givaro to matrix_modn_dense_float and matrix_modn_dense_double.

diff --git a/module_list.py b/module_list.py
--- a/module_list.py
+++ b/module_list.py
@@ -942,13 +942,13 @@
     Extension('sage.matrix.matrix_modn_dense_float',
               sources = ['sage/matrix/matrix_modn_dense_float.pyx'],
               language="c++",
-              libraries = ['gmp', 'linbox', BLAS, BLAS2],
+              libraries = ['gmp', 'linbox', 'givaro', BLAS, BLAS2],
               extra_compile_args = ['-DDISABLE_COMMENTATOR']),
 
     Extension('sage.matrix.matrix_modn_dense_double',
               sources = ['sage/matrix/matrix_modn_dense_double.pyx'],
               language="c++",
-              libraries = ['gmp', 'linbox', BLAS, BLAS2],
+              libraries = ['gmp', 'linbox', 'givaro', BLAS, BLAS2],
               extra_compile_args = ['-DDISABLE_COMMENTATOR']),
 
     Extension('sage.matrix.matrix_modn_sparse',
